daisy/step_create_disks.go

```diff
@@ -47,6 +51,10 @@ func (c *CreateDisks) run(ctx context.Context, s *Step) DError {
 		go func(cd *Disk) {
 			defer wg.Done()
 
+			if res, _ := strconv.ParseBool(cd.IsWindows); res {
+				cd.GuestOsFeatures = combineGuestOSFeatures(cd.GuestOsFeatures, "WINDOWS")
```

Comment:
This should probably be done in populate so it's validated, instead of modifying it directly in run. Otherwise we might run an invalid workflow and not know until we attempt to run CreateDisk

Reply:
sgtm